stellargraph/losses.py

```diff
@@ -20,21 +20,32 @@
 
 
 @experimental(reason="lack of unit tests", issues=[804])
-def graph_log_likelihood(y_true, y_pred):
+def graph_log_likelihood(batch_adj, wys_output):
     """
+    Computes the graph log likelihood loss function as in https://arxiv.org/abs/1710.09599. This is different
```

Comment:
Per https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings , the python convention that tooling will obey is to have a one line summary as its own paragraph:

```python
    """
    Computes the graph log likelihood loss function as in https://arxiv.org/abs/1710.09599.

    This is different ...

    ...
    """
```